paybutton/rollup.dev.js

```diff
@@ -48,8 +48,13 @@ function template ( { attributes, title, bundle } ) {
           <div class="paybutton" to="bitcoincash:qrmm7edwuj4jf7tnvygjyztyy0a0qxvl7q9ayphulp" amount="50000" currency="SAT" detect-payment="true"</div>
           <br />
           <div class="paybutton-widget" to="bitcoincash:qrmm7edwuj4jf7tnvygjyztyy0a0qxvl7q9ayphulp" amount="50000" currency="SAT" detect-payment="true"</div>
+          <br />
+          <div id="js-test" />
         </div>
       </body>
+      <script>
+        PayButton.renderWidget(document.getElementById('js-test'), { to: 'bitcoincash:qrmm7edwuj4jf7tnvygjyztyy0a0qxvl7q9ayphulp', currency: 'SAT', detect-payment: 'true' });
```

Comment:
Wasn't in scope to add this but it doesn't seem to be working either way.

Reply:
I may as well remove it as I'm going to be fixing the docs anyways.